logstash-core/lib/logstash/api/modules/base.rb

```diff
@@ -0,0 +1,41 @@
+require "logstash/api/app_helpers"
+require "logstash/api/command_factory"
+
+module LogStash
+  module Api
+    module Modules
+      class Base < ::Sinatra::Base
```

Comment:
I saw initially the modules as the resources (web api endpoints ) definition, base class should not be part of the modules as it does not really define web api resources. 

On the other side, just my experience and way of thinkings (does not have to be yours), but I like the idea of having a main app class, or file, in that case I guess we could for example move this base class to the the `logstash/api` directory, what do you think?